### PR TITLE
Create test to add banners in between gallery images.

### PR DESCRIPTION
**Pull request by JonNorman**
## What does this change?
We have loads of room between images in a gallery and we only have MPUs serving there currently.

Why not banner ads?

## What is the value of this and can you measure success?
More $$$ from those sweet, sweet banner CPMs.

## Does this affect other platforms - Amp, Apps, etc?
No.

@ScottPainterGNM:

1. What % of people should this opt into the test automatically? (This will be done in a separate PR).

2. Should the ads between the gallery images be centred?

## Tested in CODE?
No

### CODE REVIEW

**Comment by JonNorman** [comment]
[Gallery Banner Ads](https://trello.com/c/rNzlMgOw/126-gallery-banner-ads)

File: applications/app/views/fragments/gallerySlot.scala.html [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+)(implicit request: RequestHeader)
 @import conf.switches.Switches.CommercialSwitch
+@import mvt.CommercialGalleryBannerAds
+
+@additionalSizes = @{
+    if(CommercialGalleryBannerAds.isParticipating)
+        Seq("970,250", "970,350")
+    else
+        Seq.empty
+}
```
Comment (TBonnin) [inline applications/app/views/fragments/gallerySlot.scala.html]: This template has already a bunch of logic. Could it leave outside of the template code where is harder to read and test?

File: applications/app/views/fragments/gallerySlot.scala.html [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
     id: Int,
     isMobile: Boolean = false
-)
+)(implicit request: RequestHeader)
 @import conf.switches.Switches.CommercialSwitch
+@import mvt.CommercialGalleryBannerAds
+
+@additionalSizes = @{
+    if(CommercialGalleryBannerAds.isParticipating)
+        Seq("970,250", "970,350")
```
Comment (ScottPainterGNM) [inline applications/app/views/fragments/gallerySlot.scala.html]: There's no slot for 970,350. 

Participants should see one of the following: 300x250, 970,250 or 900,250 (we keep MPU's in as we can then make the bidding for slots available to high value MPU's still).

**Comment by ScottPainterGNM** [comment]
Will chat to Jamie about point 1, I think regarding point 2 that maybe keep them left aligned to where the slot starts? Have you got any screenshots of it working yet or need to deploy this so we can double check?

File: applications/app/views/fragments/gallerySlot.scala.html (hunk 1 above)
Comment (JonNorman) [inline applications/app/views/fragments/gallerySlot.scala.html]: Hey Thomas, good point...but I think I'll leave it for now as, once this test is complete (in a week), this logic will be removed and the sizes will just be added to the hard-coded string.

**Comment by ScottPainterGNM** [comment]
Point 1 - 50/50 as we want to use this as an impact measurement to present back to the business.

File: applications/app/views/fragments/gallerySlot.scala.html [hunk 3]
```diff
@@ -12,7 +20,7 @@
         @fragments.commercial.adSlot(
             slotName,
             Seq("gallery-inline", "dark") ++ (if(isMobile) Some("mobile") else None),
-            Map("mobile" -> sizes),
+            Map("mobile" -> (sizes ++ additionalSizes)),
```
Comment (regiskuckaertz) [inline applications/app/views/fragments/gallerySlot.scala.html]: You should add `additionalSizes` to the `desktop` (or even `leftCol` or `wide`) breakpoint instead, so: `Map("mobile" -> sizes) + additionalSizes`

Comment (JonNorman) [inline applications/app/views/fragments/gallerySlot.scala.html]: I don't quite follow - I'll pop over and discuss!

Comment (JonNorman) [inline applications/app/views/fragments/gallerySlot.scala.html]: Good point - I think `desktop` should suffice as that breakpoint min width is 980...

**Pull request merged**